examples/chip-tool/commands/Commands.h

```diff
@@ -0,0 +1,110 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_COMMANDS_H__
+#define __CHIPTOOL_COMMANDS_H__
+
+#include <string>
+
+#include "Echo.h"
+#include "EchoBle.h"
+#include "Off.h"
+#include "On.h"
+#include "ReadOnOff.h"
+#include "Toggle.h"
+
+// The list of available commands
+Echo echo;
```

Comment:
This is a header file. These should all be declared `extern` and ideally `const` ?

Better yet, can we avoid the globals altogether ? This approach requires static initializers.

Reply:
Fixed.